internal/rpcproxy/sshkeys.go

```diff
@@ -0,0 +1,123 @@
+// Copyright 2025 Canonical.
```

Comment:
could we rename this file to keymanagerfacade.go?

Reply:
What about just `keymanager.go`?